src/Basemo.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+contract Basemo is Initializable, OwnableUpgradeable, UUPSUpgradeable {
+    IERC20 public usdcToken;
+    uint256 private debtCounter;
+
+    struct Debt {
```

Comment:
What's the best practice for declaring structs inside of contracts? is it better to declare as an import? I honestly don't know, just asking in case this has any interference in variable declaration order and allowing upgrades